src/app/member/components/MemberDetials.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Card, CardContent, CardHeader } from "@/components/ui/card";
+import { UserAvatar } from "@/common/Avatar/UserAvatar";
+import { StatusBadge } from "@/common/StatusBadge/StatusBadge";
+import { Briefcase, MapPin, Phone, Shield, User } from "lucide-react";
+import { UserDetails } from "../details/[...slug]/page";
```

Comment:
i think it should be place in this  component and call it in page   or in `type` file if it is using in more than one file